src/audio/google/google_rtc_audio_processing.c

```diff
@@ -52,25 +52,51 @@ DECLARE_SOF_RT_UUID("google-rtc-audio-processing", google_rtc_audio_processing_u
 DECLARE_TR_CTX(google_rtc_audio_processing_tr, SOF_UUID(google_rtc_audio_processing_uuid),
 			   LOG_LEVEL_INFO);
 
-struct google_rtc_audio_processing_comp_data {
-#if CONFIG_IPC_MAJOR_4
-	struct sof_ipc4_aec_config config;
+#if !(defined(__ZEPHYR__) && defined(CONFIG_XTENSA))
+/* Zephyr provides uncached memory for static variables on SMP, but we
```

Comment:
Don;t use "align" and "switch to cache" later

Why use static  variables at all? Those may be put in module context, allocated in cached memory. Using of statics makes run several instances of AEC impossible. 
And 
- it is easy to forget arch_xtensa_cached_ptr(p) somewhere 
- it may be problematic in case of AEC as a loadable module
- most important:
``static __aligned(PLATFORM_DCACHE_ALIGN)
uint8_t aec_mem_blob[CONFIG_COMP_GOOGLE_RTC_AUDIO_PROCESSING_MEMORY_BUFFER_SIZE_BYTES];``
makes the addres of a var aligned, not a var size. Linker can put another static in the same cacheline, and that is bad. And extremely hard to track.